assets/css/app.css

```diff
@@ -34,15 +34,6 @@
   text-shadow: 3px 1px 10px rgb(22, 22, 22);
 }
 
-.ratio-video {
-  aspect-ratio: 16 / 9;
-}
-
-.video-divider {
-  height: 70px;
-  min-height: 70px;
-}
-
```

Comment:
What do these changes have to do with the toast component?

Reply:
> What do these changes have to do with the toast component?

nothing, I removed unnecessary classes when adding animations